### ISSUE TITLE
Accessibility issue: Elements must meet minimum color contrast ratio thresholds on /notebook-ci-testing/notebooks/NIRSPEC/IFU/JWPipeNB-NIRSpec-IFU.html

### ISSUE THREAD

**Issue by dulude** [spacetelescope/notebook-ci-testing#367]
## What
An accessibility scan flagged 7 elements on https://spacetelescope.github.io/notebook-ci-testing/notebooks/NIRSPEC/IFU/JWPipeNB-NIRSpec-IFU.html because elements must meet minimum color contrast ratio thresholds. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/color-contrast?application=playwright.

The following elements need attention:

- `<a class="reference external" href="https://jwst-docs.stsci.edu/jwst-science-calibration-pipeline-overview">JDox</a>` (selector: `#package-imports > .alert-block.alert-info.alert > p > .external.reference:nth-child(1)`)
- `<a class="reference external" href="https://jwst-pipeline.readthedocs.io">ReadtheDocs</a>` (selector: `.alert-block.alert-info.alert > p > .external.reference:nth-child(2)`)
- `<a class="reference external" href="https://github.com/spacetelescope/jwst">Github</a>` (selector: `.alert-block.alert-info.alert > p > .external.reference:nth-child(3)`)
- `<a class="reference external" href="https://astroquery.readthedocs.io/en/latest/index.html">astroquery</a>` (selector: `#demo-mode-setup-ignore-if-not-using-demo-data > .alert-block.alert-info.alert > p > .external.reference`)
- `<a class="reference external" href="https://jwst-docs.stsci.edu/known-issues-with-jwst-data/shower-and-snowball-artifacts#gsc.tab=0">snowballs</a>` (selector: `.alert-info.alert:nth-child(9) > p > .external.reference`)
- `<a class="reference external" href="https://github.com/spacetelescope/jdat_notebooks/tree/main/notebooks/NIRSpec/NIRSpec_NSClean">here</a>` (selector: `#configure-spec2pipeline > .alert-info.alert > p > .external.reference`)
- `<a class="reference external" href="https://jwst-pipeline.readthedocs.io/en/latest/jwst/outlier_detection/outlier_detection_ifu.html#outlier-detection-ifu">here</a>` (selector: `#configure-spec3pipeline > .alert-info.alert > p > .external.reference`)

To fix this, ensure the contrast between foreground and background colors meets wcag 2 aa minimum contrast ratio thresholds.

Specifically:

Fix any of the following:
-   Element has insufficient color contrast of 4.1 (foreground color: #176de8, background color: #cff4fc, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.2 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
